src/client/styling/pokedex.css

```diff
@@ -26,8 +26,8 @@
     text-align: center;
     padding: 0.5rem;
     grid-template-columns: 1fr;
-    box-shadow: 0px 0px 10px rgb(var(--clr-shadow));
-
+    box-shadow: -2px -2px 5px rgba(var(--clr-pokeblue), 0.5), 
+                2px 2px 5px rgba(var(--clr-pokeorange), 0.5)
```

Comment:
Pretty :)

Reply:
Oooo - css variables - nice :)

Reply:
> Oooo - css variables - nice :)

If you like those, wait till you see my wordle clone ;)

Reply:
@glowkeeper If you like those, wait till you see my wordle clone ;)